packages/html-reporter/src/testCaseView.css

```diff
@@ -35,11 +35,23 @@
 }
 
 .test-case-title {
+  --title-size: 32px !important;
   flex: none;
   padding: 8px;
   font-weight: 400;
-  font-size: 32px !important;
+  font-size: var(--title-size);
   line-height: 1.25 !important;
+  cursor: pointer;
+}
+
+.test-case-title .octicon {
+  width: var(--title-size);
```

Comment:
It is unusual to use the same values for the width / height and the font size.

Reply:
Yeah I know, but this was the best solution I could figure out to ensure that the icon matches the font size if someone changes it in the future.

Reply:
These values can surely depend on the title size, but new variables should be added for the icon for consistency.

```css
--title-size: 32px !important;

--octicon-width: var(--title-size);
--octicon-height: var(--title-size);
```